app/pb_kits/playbook/pb_currency/_currency.jsx

```diff
@@ -1,72 +1,75 @@
 /* @flow */
 
-import React from 'react'
-import classnames from 'classnames'
+import React from "react";
+import classnames from "classnames";
 
-import {
-  Body,
-  Caption,
-  Title,
-} from '../'
+import { Body, Caption, Title } from "../";
 
 type CurrencyProps = {
-  align?: 'left' | 'center' | 'right',
+  align?: "left" | "center" | "right",
   className: String,
-  currencySymbol: '$',
+  currencySymbol: "$",
   label?: String,
-  separator?: '.' | ',',
-  size?: 'sm' | 'lg',
+  separator?: "." | ",",
+  size?: "sm" | "lg",
   value?: String,
-  unit?: '00',
-}
+  unit?: "00"
+};
 
-const kitCSS = ({align='left'}: CurrencyProps) => {
-  let css = 'pb_currency_kit'
-  css += `_${align}`
-  return css
-}
+const kitCSS = ({ align = "left" }: CurrencyProps) => {
+  let css = "pb_currency_kit";
+  css += `_${align}`;
+  return css;
+};
 
-const bodyCSS = ({}: CurrencyProps) => {
-  let css = ''
-  return css
-}
+const bodyCSS = ({  }: CurrencyProps) => {
+  let css = "";
+  return css;
+};
 
-const symbolCSS = (currencySymbol) => {
+const symbolCSS = currencySymbol => {
   return classnames({
-    dollar_sign: currencySymbol === '$',
-  })
-}
+    dollar_sign: currencySymbol === "$"
+  });
+};
 
 const sizes = {
   lg: 1,
-  sm: 2,
-}
+  sm: 2
+};
 
 const Currency = (props: CurrencyProps) => {
   const {
-    currencySymbol='$',
-    label='',
-    separator='.',
-    size='sm',
-    units='00',
-    value,
-  } = props
+    currencySymbol = "$",
+    label = "",
+    separator = ".",
+    size = "sm",
+    units = "00",
+    value
+  } = props;
 
   return (
     <div className={kitCSS(props)}>
       <Caption>{label}</Caption>
-      <div className='pb_currency_wrapper'>
-        <Body className={symbolCSS(currencySymbol)}>{currencySymbol}</Body>
-        <Title
-            className='pb_currency_value'
+      <div className="pb_currency_wrapper">
+        <Body className={symbolCSS(currencySymbol)}
+            color={"light"}
+        >
+          {currencySymbol}
+        </Body>
+        <Title className="pb_currency_value"
             size={sizes[size]}
         >
           {`${value}${separator}`}
         </Title>
-        <Body className='unit'>{units}</Body>
+        <Body className="cents"
+            color={"light"}
+        >
+          {units}
```

Comment:
See comment above. Prop name is `unit`, this should be consistent with prop name. (units)